.pre-commit-config.yaml

```
@@ -0,0 +1,56 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.1.0
+  hooks:
+  - id: check-added-large-files
+  - id: check-toml
+  - id: check-yaml
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
```

```suggestion
  - id: trailing-whitespace
    args: [--markdown-linebreak-ext=md]
```
So that `.md` files do not have trailing whitespace trimmed as this could be used for line breaks.